xls/modules/rle/rle_interface.x

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 The XLS Authors
```

Comment:
maybe rename this to `rle_common.x` to match the convention in https://github.com/google/xls/tree/main/xls/modules/aes?